docs/development.md

```diff
@@ -99,6 +99,16 @@ $ cd $GOPATH/src/k8s.io/ingress-nginx
 $ make e2e-test
 ```
 
+To run unit-tests for lua code locally, run:
+
+```console
+$ cd $GOPATH/src/k8s.io/ingress-nginx
+$ ./rootfs/etc/nginx/lua/test/up.sh
+$ make lua-test
+```
+
+Lua tests are located in `$GOPATH/src/k8s.io/ingress-nginx/rootfs/etc/nginx/lua/test`. When creating a new test file it must follow the naming convention `<mytest>-test.lua` or it will be ignored. 
```

Comment:
can we enforce `_test` pattern rather than `-`?

Reply:
Unfortunately no. Initially this is the pattern I used. However, this causes errors because these files are converted to go source code in the bindata.go file and underscores are not allowed in go naming convention. It was causing the ` ./hack/verify-bindata.sh` test to fail.

Reply:
Hmm, are you sure? Looking at https://github.com/kubernetes/ingress-nginx/blob/master/hack/verify-gofmt.sh#L32 `bindata.go` should be checked.